stable/chaoskube/README.md

```diff
@@ -65,4 +68,7 @@ $ helm install \
     - --interval=10m
```

Comment:
Could you drop the `--in-cluster` flag as well? :blush: It doesn't exist anymore.